### ISSUE TITLE
[$80] Empty page and console error when click view previous interviews

### ISSUE THREAD

**Issue by sdgun** [topcoder-platform/taas-app#544]
**Steps**

- Select 'Schedule interview' action from CANDIDATES TO REVIEW page for a candidate who has multiple interview rounds scheduled. example candidate [here](https://platform.topcoder-dev.com/taas/myteams/18961/positions/da5e3de0-4937-484f-869b-10b628bfd0bd/candidates/interviews)
- Click Actions drop down and select 'View previous interviews'

Empty page and console error when click view previous interviews

https://user-images.githubusercontent.com/4592712/139382577-14d45196-3276-40b6-8147-ff579c1f11ad.mp4

**Comment by yoution** [comment]
@maxceem please assign to me

**Comment by maxceem** [comment]
@yoution unassigned

**Comment by yoution** [comment]
@maxceem please assign to me

**Comment by maxceem** [comment]
@yoution assigned

**Comment by yoution** [comment]
@maxceem in order to get the email info, I copy the code https://github.com/topcoder-platform/taas-apis/blob/dev/src/services/NotificationsSchedulerService.js#L83
but I found the host detail info will be broken
![image](https://user-images.githubusercontent.com/3863914/139538916-5a6ff71e-254d-485d-b6c3-23364d42d944.png)

**Comment by maxceem** [comment]
@yoution hmm, maybe that user data is broken on DEV.

I also check this issue for my user `maxceem-dev`, still get this issue but I believe DEV data should be good. https://monosnap.com/file/HH4t6flgGZIXGqSQj1ZdThwdMqXGqV

Could you try it with my use `maxceem-dev` on this page? https://platform.topcoder-dev.com/taas/myteams/16786/positions/c1f3e159-729e-4eae-a9db-4f445a37e968/candidates/interviews

Are you still getting `undefined` for `userId`?

**Comment by yoution** [comment]
@maxceem please assign to me

**Comment by yoution** [comment]
@maxceem please review

**Comment by yoution** [comment]
@maxceem you said you will increase to $80

**Comment by maxceem** [comment]
@sdgun fixed and ready for QA.

Instead of emails we are showing user full names because we don't allow adding additional users to interview by emails. And in all other places we are showing attendees full names. Also, it feels more private then exposing user emails.

Actually, it seems that this feature of see history is not that valuable anymore, as we would always have the same 2 users in the interview. The only time would be different.

![image](https://user-images.githubusercontent.com/146016/139822262-98890c66-3efc-4f76-9f43-f95a2a1aea8c.png)

**Comment by maxceem** [comment]
yes, sure @yoution, increased in the ticket, I would process payment as soon as QA is done.

**Comment by maxceem** [comment]
@sdgun just in case, this is ready for QA as per https://github.com/topcoder-platform/taas-app/issues/544#issuecomment-957274680

**Comment by sdgun** [comment]
Verified in Dev.

![image](https://user-images.githubusercontent.com/4592712/140021612-b745cc0e-f237-41ff-9279-76394acddc83.png)

**Comment by maxceem** [comment]
Challenge https://www.topcoder.com/challenges/6f21a086-43bf-451b-b56e-4dd083ec08e1 has been created for this ticket.<br/><br/>```This is an automated message for maxceem via Topcoder X```

**Comment by maxceem** [comment]
Challenge https://www.topcoder.com/challenges/6f21a086-43bf-451b-b56e-4dd083ec08e1 has been assigned to yoution.<br/><br/>```This is an automated message for maxceem via Topcoder X```

**Comment by maxceem** [comment]
Payment task has been updated: https://www.topcoder.com/challenges/6f21a086-43bf-451b-b56e-4dd083ec08e1
*Payments Complete*
Winner: yoution
Copilot: maxceem
Challenge `6f21a086-43bf-451b-b56e-4dd083ec08e1` has been paid and closed.<br/><br/>```This is an automated message for maxceem via Topcoder X```
